draft-ietf-core-senml.md

```diff
@@ -416,6 +416,14 @@ to describe that a resource is available in a SenML format in the
 first place. The relevant media type indicator is included in the
 Content-Type (ct=) attribute.
 
+## Configuration and Actuation usage
+
+SenML can also be used for configuring parameters and controlling
+actuators. When a SenML Pack is sent (e.g., using a HTTP/CoAP POST or
+PUT method) and the semantics of the target are such that SenML is
+interpreted as configuration/actuation, SenML Records are interpreted
+as a request to change the values of given resources (names) to given
```

Comment:
We could say "resources or subresources (as identified by their names)" (the names do not always identify actual separate resources).